### ISSUE TITLE
Modular backup

### ISSUE THREAD

**Issue by serrq** [z-huang/music#1517]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Some users asked to integrate downloaded items in the backup. I don't want a heavy backup. I want a skinny backup, just playlists, songs, albums, etc. in plain text (just header, no body).

To do this, an user can building a tailored backup according with its needs.

So modular backup is necessary.

### Why do you want this feature?

Each user has different needs.

![modular-backup](https://github.com/user-attachments/assets/cef80bb9-0f26-4e65-8071-276a7f785950)

**Comment by serrq** [comment]
#1436

**Comment by Anonuser-code** [comment]
Yes. I think most of users need back up their playlist. We should be able restrore songs even if we delete the app or change the phone for any reason.
